### ISSUE TITLE
Translations for AdNauseam website?

### ISSUE THREAD

**Issue by cqx931** [dhowe/AdNauseam#1656]
Helen briefly mentioned it during our meeting today.

It won't be too much trouble if we want to do it. I just need to set up the websites in a way that we can use various language tags and upload the text to Crowdin.

It will just be the main text though, the banner images will still be in English.

**Comment by mushon** [comment]
I see no reason why not. Do you need anything from me to make it happen?

**Comment by cqx931** [comment]
@mushon No, as long as everyone agrees, I will work on it to make it happen :).

Website text file uploaded to Crowdin.
https://crowdin.com/project/adnauseam/settings#files

**Comment by cqx931** [comment]
Up to this point, the website has now been translated to Chinese, Finnish and Spanish with help from the crowd-in community. 
https://crowdin.com/project/adnauseam/settings#translations
